mutagen/mp4/__init__.py

```diff
@@ -891,6 +892,119 @@ def to_line(key, value):
         return u"\n".join(values)
 
 
+class MP4Chapters(object):
```

Comment:
What about using collections.abc.Sequence?

Any maybe instead of tuples use Chapter() objects so we can extend them later on.

Reply:
Implemented MP4Chapters as collections.abc.Sequence with `__getitem__` and `__len__`.

The Chapter class is currently not much more than a namedtuple.